docs/reference/logging/overview.md

```diff
@@ -41,6 +43,21 @@ Logs that reach `max-size` are rolled over. You can set the size in kilobytes(k)
 
 If `max-size` and `max-file` are set, `docker logs` only returns the log lines from the newest log file.
 
+The `labels` and `env` options adds additional attributes to be logged
+when using this logging driver. Each of these options takes a list of
+key separated by comma. A additional field appears for containers that
+have a environment variable or label matching these keys.
+
+Giving the docker daemon is started with the options
+`--log-driver=json-file --log-opt labels=foo --log-opt env=foo,fizz`,
+running a container with `--label foo=bar -e fizz=buzz` adds the
+`attrs` field that looks like:
+
+    "attrs":{"fizz":"buzz","foo":"bar"}
```

Comment:
Note clear how I would get this to appear `docker inspect` or something else.  I'm assuming `inspect`